docs/configuration.rst

```diff
@@ -0,0 +1,26 @@
+Configuration Reference
+=======================
+
+``towncrier`` has many knobs and switches you can use, to customise it to your project's needs.
+The setup in the `Quick Start <quickstart.html>`_ doesn't touch on many, but this document will detail each of these options for you!
+
+For how to perform common customisation tasks, see `Customisation <customisation/index.html>`_.
+
+``[tool.towncrier]``
+--------------------
+
+All configuration for ``towncrier`` sits inside ``pyproject.toml``, under the ``tool.towncrier`` namespace.
```

Comment:
Consider also mentioning PEP-518  to explain why the config is in a file named `pyproject.toml`.